Source/JavaScriptCore/bytecode/StructureStubInfo.h

```diff
@@ -695,19 +709,17 @@ class SharedJITStubSet {
         return nullptr;
     }
 
-    RefPtr<PolymorphicAccessJITStubRoutine> getMegamorphic(AccessType) const;
-    void setMegamorphic(AccessType, Ref<PolymorphicAccessJITStubRoutine>);
+    using StatelessCacheKey = std::tuple<AccessType, AccessCase::AccessType, bool, bool, bool, bool>;
```

Comment:
It was actually hard to see it in enum for all of these bools in various places. I feel like using bool is better here since this is not randomly used key. It is only created from StructureStubInfo, for InlineCacheCompiler.
If this is used and created in various places, I think enum would be good, or using struct here would be nice. But given that it is only used in one place, I think this is simpler and better (plus, better for extensibility).